.vscode/launch.json

```diff
@@ -41,7 +41,17 @@
 			],
 			"outFiles": ["${workspaceFolder}/out/test/**/*.js"],
 			"preLaunchTask": "npm: watch"
-		},
+        },
+        {
+            "type": "node",
+            "name": "Extension Unit Tests (Jest)",
```

Comment:
If we have both tests here, I think it would be better to keep them relevant to each other:
We can name them:
a. Extension Tests (Mocha)
b. Extension Tests (Jest)
Or:
a. Extension Tests (vscode)
b. Extension Tests (no-vscode)
To keep it also in sync with `npm scripts`

Reply:
I changed to the second option...